frontend/src/client/Call/ListCalls.js

```diff
@@ -26,7 +26,12 @@ class ListCalls extends React.Component {
     super(props);
 
   }
-
+  
+  
+  //https://stackoverflow.com/questions/33188994/scroll-to-the-top-of-the-page-after-render-in-react-js
+  useEffect(() => {
```

Comment:
It looks like you're calling this in the class body. Consider adding it to the render override.

But expanding on this, currently it triggers once. If you move it to the render call, it will be called every time a property or the virtual DOM above changes.  This is not a great behavior. You could be scrolling to find past entry and a new one pops and now you're at the top.

I could see this implemented in such a way where it only gets called if you're already at the top of the page.

Reply:
Great points, thanks for the feedback! I'll see if I can move this into render and limit the cases where it pulls up to the top.

Reply:
I believe the approach I have now should be improved to now 1) occur after the `render` call with `componentDidMount` and 2) take into consideration if the user is not at the top of the page

Reply:
What's inside `componentDidMount` should go in 'render' (make sure to check that the ref is not undefined though).

And I think your current implementation still has some bugs. Following up on another comment.